src/Arcus.Templates.Tests.Integration/Health/v1/HealthDockerEndpointTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Arcus.Templates.Tests.Integration.Fixture;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Templates.Tests.Integration.Health.v1
+{
+    [Collection("Docker")]
+    public class HealthDockerEndpointTests
+    {
+        private readonly TestConfig _configuration;
+        private readonly ITestOutputHelper _outputWriter;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HealthDockerEndpointTests"/> class.
+        /// </summary>
+        public HealthDockerEndpointTests(ITestOutputHelper outputWriter)
+        {
+            _configuration = TestConfig.Create();
+            _outputWriter = outputWriter;
+        }
+
+        [Fact]
+        public async Task Health_Get_Docker_Succeeds()
```

Comment:
Thinking out loud - Why is this test different from the non-Docker one and can we make it so that we re-use the same test for different hosting options?

Let's discuss first before changing everything btw 😁

Reply:
It's different because in the Docker test; we don't have to spin up anything (since the Docker container is running).
I think that we should leave the current YAML build and this test so we know we can run this in a Docker container; but beside this we should have a lot of others to test the project options and combintation of options.

Reply:
In that case, would it make sense to use another test category for this?

Reply:
I thought that I added a `[Category]` attribute on this class. Yes.

Reply:
Ah I see now but we have not made any different in the release pipeline.

Would it make sense to run the categories seperately? With that I mean one stage called "Integration Tests" with 2 jobs: 1 for Docker-based tests, 1 for the integration tests without any infra dependency?

Reply:
Yeah, that makes sense. Both the release and CI pipeline then?

Reply:
I'd say yes

Reply:
Added #44 to refactor the builds to reduce duplication

Reply:
Ok! I'll pick this up tomorrow

Reply:
If you want to split it in a new issue that's also fine, but we have time. Just don't want to block the PR on this.

Reply:
Ah, OK; then maybe in a diff issue.
It's not that the tests are not run; so maybe better other issue :).

Reply:
Created #46